### ISSUE TITLE
Highlight outliers for GC_Dropout

### ISSUE THREAD

**Issue by alieden** [Clinical-Genomics/statina#212]
Since we've recently seen more batches with higher than expected GC_Dropout the usage of a upper threshold value has become difficult to use. In theory we believe that the current batch specific normalization procedure should be able to handle batch effects like this and this is supported by preliminary data analysis. In relation to this we would now like to have an aid in the determination of failed samples i.e. in the GCD and ATD boxplots it would be good if outliers in the data (boxplots) could be highlighted some way?

**Comment by mayabrandi** [comment]
Hej! 

Can we have a chat about this @alieden ? 

In my world its easy to see the outliers in the plots so maybe I'm misunderstanding.. We better talk!

![Skärmavbild 2022-02-14 kl  09 59 02](https://user-images.githubusercontent.com/1306333/153832530-677790b6-77fc-41a1-9c4f-4eb6924090c0.png)

**Comment by alieden** [comment]
Yes it's probably just easier to explain in our internal supporting text for the analysis. I had in my memory that there were cases when outliers could end up within the min-max but I couldn't find that now. Maybe I just mixed it up with something else. I'll close the issue.
